### ISSUE TITLE
presence of "init.lua" causes a segfault when using a Lua filter

### ISSUE THREAD

**Issue by novelistparty** [jgm/pandoc#5510]
> macOS 10.14.5, pandoc 2.2.1, downloaded from pandoc.org

Without an `init.lua` in my data directory, the following works just fine:
```
pandoc --lua-filter substitute.lua <(echo "{{hello world}}")
```
, where the contents of `substitute.lua` are 
```
return {
  {
    Str = function (elem)
      if elem.text == "{{helloworld}}" then
        return pandoc.Emph {pandoc.Str "Hello, World"}
      else
        return elem
      end
    end,
  }
}
```

If I place even an *empty* `init.lua` in my data directory, the same command causes pandoc to exit with code 139, and says 
```
Segmentation fault: 11
```

Any help is welcome. I'm trying to play with filters and would like to place some Lua initialization in init.lua. Thanks!

**Comment by jgm** [comment]
@tarleb would know about this...

**Comment by tarleb** [comment]
This is a bug that was fixed in pandoc 2.7. Please update pandoc. If that is not possible, create `init.lua` by running

    pandoc --print-default-data-file init.lua > ~/.pandoc/init.lua
